projects/fsastorefrontlib/src/cms-components/b2b/unit/assignments/cells/remove-product-cell/remove-product-cell.component.ts

```diff
@@ -36,6 +41,18 @@ export class RemoveProductCellComponent extends CellComponent
       this.model.assignmentCode,
       parentUintId
     );
+    this.notify(this.model, this.DEASSIGNED_STATE);
+  }
+
+  protected notify(item, state) {
```

Comment:
Method is not publicly accessible in CellComponent class, that's why we need to copy it in both components. Another way would be to move the method to service but that would require the creation of our` ProductAssignmenListService extends ListService< FSProductAssignment > ` which would I avoid as we have some differences in implementation.
If they rise access modifier to public in future release, we can just remove both of them :)